File: java-main/java-main/Array/p23.java
class Demo{
	void fun(int arr[]){
		for(int x : arr){
			System.out.println(x);
		}
		for(int i=0;i<arr.length;i++){
			arr[i]=arr[i]+50;
		}
	}
	public static void main(String[]args){
		Demo obj = new Demo();
		int arr[]={50,100,150};
		obj.fun(arr);
		for(int x : arr){
			System.out.println(x);
		}
	}
}

File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/struct/p23.c
#include<stdio.h>
struct Demo{

	int x;
	float y;
};
void fun(struct Demo obj){

	printf("%d\n",obj.x);
	printf("%f\n",obj.y);
}
void gun(struct Demo *ptr){
	printf("%d\n",ptr->x);
	printf("%f\n",(*ptr).y);
}
void main(){

	struct Demo obj={10,20.0f};
	fun(obj);
	gun(&obj);
}

